src/os2datascanner/projects/admin/adminapp/views/miniscanner_views.py

```diff
@@ -36,25 +36,20 @@ def get_context_data(self):
 
         return context
 
-
-def execute_mini_scan(request):  # noqa:CCR001
-    context = {
-        "file_obj": (file_obj := request.FILES.get("file")),
-        "raw_rule": (raw_rule := request.POST.get("rule")),
-        "halfbaked_rule": (halfbaked_rule := json.loads(raw_rule or "null")),
-
-        "replies": (replies := []),
-    }
-
-    rule = None
-    if halfbaked_rule:
-        rule = Rule.from_json_object(halfbaked_rule)
-
-    if file_obj and rule:
-        with NamedTemporaryResource(file_obj.name) as ntr:
+def mini_scan(item, rule):
+    try:
+        name = item.name
+    except:
```

Comment:
Consider declaring what exception is expected ie. `except ExceptionNameHere:`

Reply:
Done ! Also added a second except, in case of an unexpected error. Marking as resolved and pushing updated code very soon !